src/entrypoint/graphql-core-server.ts

```diff
@@ -199,8 +171,8 @@ const resolvers = {
     updateUser: (_, __, { wallet }) => ({
       setUsername: async ({ username }) => await wallet.setUsername({ username }),
       setLanguage: async ({ language }) => await wallet.setLanguage({ language }),
-      updateUsername: async (input) => await wallet.updateUsername(input),
-      updateLanguage: async (input) => await wallet.updateLanguage(input),
+      updateUsername: (input) => wallet.updateUsername(input),
```

Comment:
Why are we using a mixed pattern of using await for some functions but not for others. For example line #172 uses `await` but this line has it removed.